.github/actions/compile/action.yml

```diff
@@ -35,19 +35,11 @@ runs:
       with:
         dotnet-version: 5.x
 
-    - name: Install Dependencies
-      if: inputs.dependency-identifiers != '' || inputs.msbuild-dependency-target != ''
-      uses: KSPModdingLibs/KSPBuildTools/.github/actions/install-dependencies@main
-      with:
-        dependency-identifiers: ${{ inputs.dependency-identifiers }}
-        msbuild-dependency-target: ${{ inputs.msbuild-dependency-target }}
-        solution-file-path: ${{ inputs.solution-file-path }}
-
     - name: Download KSP Libs
       shell: bash
       run: |
         wget --quiet ${{ inputs.ksp-zip-url }} -O /tmp/ksp.zip
-        unzip -q -P "${{ inputs.ksp-zip-password }}" /tmp/ksp.zip -d "${{ env.KSP_ROOT }}"
+        unzip -q -P "${{ inputs.ksp-zip-password }}" /tmp/ksp.zip -d "${{ env.KSPRoot }}"
```

Comment:
let's revert the KSP_ROOT change and just add that when looking for defaults in KSPCommon.props

Reply:
Done